src/pattern_library/css/global/global-styles.css

```diff
@@ -0,0 +1,381 @@
+}
+
+dt + dd {
+	margin-block-start: var(--space-xs);
+}
+
+dd + dt {
+	margin-block-start: var(--space-s);
+}
+
+dd {
+	margin-inline-start: 1.5ch;
+}
+
+blockquote {
+	margin-inline: 0;
+	color: var(--color-light);
+}
+
+blockquote footer {
+	margin-block-start: var(--space-s);
+}
+
+blockquote q {
+	font-style: normal;
+}
+
+q {
+	display: inline-block;
+	font-style: italic;
+}
+
+q:before {
+	content: "\201C";
+}
+
+q:after {
+	content: "\201D";
+}
+
+:is(video, iframe[src*='youtube'], iframe[src*='vimeo']) {
+	display: block;
+	width: 100%;
+	height: auto;
+	aspect-ratio: 16/9;
+}
+
+img {
+	height: auto;
+	max-width: 100%;
+	display: block;
+}
+
+figure {
+	margin-inline: var(--space-s);
+}
+
+figcaption {
+	padding-block-start: 0.5em;
+	font-size: var(--size-step-0);
+	font-family: monospace;
+}
+
+table {
+	border: var(--stroke);
+	border-collapse: collapse;
+	width: 100%;
+}
+
+th {
+	text-align: left;
+	font-weight: var(--font-bold);
+	line-height: var(--leading-fine);
+}
+
+thead th {
+	padding-block: var(--space-s);
+}
+
+td,
+th {
+	padding: var(--space-xs) var(--space-s);
+}
+
+th:not(:only-of-type) {
+	border-block-end: var(--stroke);
+}
+
+th:only-of-type {
+	border-inline-end: var(--stroke);
+}
+
+:is(th, td) ~ :is(th, td) {
+	border-inline-start: var(--stroke);
+}
+
+tr + tr :is(th, td) {
+	border-block-start: var(--stroke);
+}
+
+caption {
+	caption-side: bottom;
+	margin-block-start: var(--space-s);
+}
+
+a:not([class]) {
+	color: var(--color-blue-600);
+}
+
+a:not([class]):hover {
+	color: var(--color-grey-800);
+}
+
+:is(h1, h2, h3, h4) a:not([class]) {
+	text-decoration-thickness: 0.1ex;
+	text-underline-offset: 0.2ex;
+}
+
+:is(h1, h2, h3, h4) a:not([class]):hover {
+	text-underline-offset: 0.3ex;
+}
+
+:focus {
+	outline: none;
+}
+
+:focus-visible {
+	outline: 2px solid var(--focus-color, currentColor);
+	outline-offset: var(--focus-offset, 0.2lh);
+}
+
+/* Firefox is the only browser that doesn't outline the whole element unless you make it display: inline-block. That in itself causes multiple flow issues so we can detect it, using its own vendor prefix and reduce focus offset
+*/
+@supports (-moz-appearance: none) {
+	:root {
+		--focus-offset: 0.08em;
+	}
+}
+
+:target {
+	scroll-margin-block: 5lh;
+}
+
+::selection {
+	color: var(--color-black);
+	background: var(--color-blue-200);
+}
+
+/* TODO: Think about how to handle margin bottom in a flow element */
+hr {
+	border: none;
+	border-block-start: var(--hr-stroke, var(--stroke));
+	margin-block: var(--flow-space, var(--space-xl));
+}
+
+svg {
+	flex-shrink: 0;
+}
+
+svg[role='img'][width][height] {
+	width: revert;
+	height: revert;
+	background: var(--color-light);
+	padding: var(--space-xs);
+}
+
+/* There's a .flow composition, but this prevents forms from having to have that applied where markup is harder to control in certain systems. It still uses the --flow-space variables though to make managing space easier */
+form > * + * {
```

Comment:
I need some explanation on the benefit of this rule - I don't understand your comment and have this nagging feeling that thus rule will then cause override rules for "special" forms.

Reply:
It auto-adds flow styles to forms, it's from the course and I haven't created any form patterns yet so I haven't investigated it.